src/Apps/Order/Utils/shippingUtils.ts

```diff
@@ -128,15 +127,22 @@ export const convertShippingAddressForExchange = (
 }
 
 export const convertShippingAddressToMutationInput = (
-  address: SavedAddressType
-): UserAddressAttributes => {
-  return omit(
-    {
-      ...address,
-      name: address?.name || "",
+  address: SavedAddressType | null
```

Comment:
we could consider importing `SettingsShippingAddress_address$data `and adding it to potential types here `address: SavedAddressType | SettingsShippingAddress_address$data | null `

Reply:
thank you!